cabal-install/src/Distribution/Client/ProjectPlanOutput.hs

```diff
@@ -199,7 +199,7 @@ encodePlanAsJson distDirLayout elaboratedInstallPlan elaboratedSharedConfig =
                     J.object $
                       [ comp2str c
                         J..= J.object
-                          ( [ "depends" J..= map (jdisplay . confInstId) (map fst ldeps)
+                          ( [ "depends" J..= map (jdisplay . confInstId) (map (\(p, _, _) -> p) ldeps)
```

Comment:
Seems like we should include the whether a dependency is private or not in the plan.json?

Reply:
Fixed.